component/vshn_postgres.jsonnet

```
@@ -901,6 +901,24 @@ local prometheusRule = {
                   },
                 ],
               },
+              {
+                name: 'postgresql-connections',
+                rules: [
+                  {
+                    alert: 'PostgreSQLConnectionsCritical',
+                    annotations: {
+                      description: 'The connections to {{ $labels.pod }} has been over 90 for 2 hours.\n  Please reducde the load of this instance.',
+                      // runbook_url: 'TBD',
+                      summary: 'Connection usage critical',
```

```suggestion
                      summary: 'Connection over limit critical',
```
Can I propose this? you can decide wether to commit.